src/test/java/fi/metatavu/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -2,76 +2,351 @@
 
 public class TestPtvConsts {
   
-  public static final String[] ORGANIZATIONS = {
-    "ae2682d3-6238-4019-b34f-b078c5f9bb50",
-    "f3cb50b9-e97f-4c00-a9c7-04265bdcadb6",
-    "9355a207-efd3-4cfb-a02b-67187f34c822"
+  public final static String[] ORGANIZATIONS = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)